control/control.leanos.xml

```diff
@@ -646,6 +649,10 @@ Please visit us at http://www.suse.com/.
                     <enable_next>yes</enable_next>
                 </module>
                 <module>
+                    <name>offline_migration</name>
+                    <enable_back>yes</enable_back>
+                </module>
+		<module>
                     <label>Product License</label>
                     <name>product_upgrade_license</name>
```

Comment:
Hm, I think we really need to first display the license. If the user declines the license we have already upgraded the product on the SCC and modified the repository setup. The rollback is not trivial and might fail.

Displaying the license works in the manual installation, I cannot remember which changes we exactly did there but I think it should be possible also here.